spec/models/fee_scheme_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe FeeScheme, type: :model do
-  let!(:lgfs_scheme_nine) { FeeScheme.find_by(name: 'LGFS', version: 9) || create(:fee_scheme, :lgfs_nine) }
-  let!(:agfs_scheme_nine) { FeeScheme.find_by(name: 'AGFS', version: 9) || create(:fee_scheme, :agfs_nine) }
-  let!(:agfs_scheme_ten) { FeeScheme.find_by(name: 'AGFS', version: 10) || create(:fee_scheme) }
+  before { seed_fee_schemes }
+
+  let(:lgfs_scheme_nine) { FeeScheme.find_by(name: 'LGFS', version: 9) }
+  let(:agfs_scheme_nine) { FeeScheme.find_by(name: 'AGFS', version: 9) }
+  let(:agfs_scheme_ten) { FeeScheme.find_by(name: 'AGFS', version: 10) }
 
```

Comment:
Shouldn't these be deleted if you are running seed_fee_schemes?

Reply:
those are are memoized vars used in the tests below as shortcuts but, no you mention, I am not sure that memoizing something that is seeded/created for each example is not a problem of itself? any idea

Reply:
I suppose it's running find_by so it's probably okay... 🤷‍♂️